pipelines/pipeline_katello_lbproxy.yml

```diff
@@ -0,0 +1,238 @@
+- hosts: localhost
+  vars:
+    forklift_name: pipeline-katello-lbproxy
+    forklift_boxes:
+      katello01:
+        box: centos7
+        memory: 6144
```

Comment:
got OOM with 6144, retrying with 8192

Reply:
interesting, as 6144 is also what we use in our other pipelines. changed to 8192 (and lowered haproxy and client to 2048)